### ISSUE TITLE
No loss 

### ISSUE THREAD

**Issue by angelusualle** [nlp-with-transformers/notebooks#133]
## Information

The problem arises in chapter:

* [ ] Introduction
* [ x] Text Classification
* [ ] Transformer Anatomy
* [ ] Multilingual Named Entity Recognition
* [ ] Text Generation
* [ ] Summarization
* [ ] Question Answering
* [ ] Making Transformers Efficient in Production
* [ ] Dealing with Few to No Labels
* [ ] Training Transformers from Scratch
* [ ] Future Directions

## Describe the bug
I ran the 
```from transformers import Trainer

trainer = Trainer(model=model, args=training_args,
                  compute_metrics=compute_metrics,
                  train_dataset=emotions_encoded["train"],
                  eval_dataset=emotions_encoded["validation"],
                  tokenizer=tokenizer)
trainer.train();
```
section, and got an error complaining about no loss in the outputs:
```
The model did not return a loss from the inputs, only the following keys: logits. For reference, the inputs it received are input_ids,attention_mask.
```

 Turns out AutoModelForSequenceClassification was expecting labels, not label. so I relabeled:
```
def relabel(batch):
    return {'labels': batch["label"]}
emotions_encoded = emotions_encoded.map(relabel)
emotions_encoded.column_names
```
